configure

```diff
@@ -1220,6 +1219,11 @@ def configure_intl(o):
         pprint.pformat(icu_config, indent=2) + '\n')
   return  # end of configure_intl
 
+def configure_inspector(o):
+  o['variables']['v8_inspector'] = b(not options.without_inspector
+                                         and o['variables']['v8_enable_i18n_support']
+                                         and o['variables']['node_use_openssl'] == 'true')
```

Comment:
Can you use `options` here?  That way it won't depend on the order in which the configure_intl an/configure_openssl/configure_inspector functions are executed.

Also, please keep lines <= 80 columns.